mm2src/coins/my_tx_history_v2.rs

```diff
@@ -274,7 +275,8 @@ impl HttpStatusCode for MyTxHistoryErrorV2 {
             MyTxHistoryErrorV2::StorageIsNotInitialized(_)
             | MyTxHistoryErrorV2::StorageError(_)
             | MyTxHistoryErrorV2::RpcError(_)
-            | MyTxHistoryErrorV2::NotSupportedFor(_) => StatusCode::INTERNAL_SERVER_ERROR,
+            | MyTxHistoryErrorV2::NotSupportedFor(_)
```

Comment:
`NotSupportedFor` is sounds more like either `404 NotFound` or `400 BadRequest` to me rather than 5xx status code :)